### ISSUE TITLE
acls with pattern type set to prefixed is not visible in acl ui

### ISSUE THREAD

**Issue by atoom** [cloudhut/kowl#142]
Hi,

Thanks for a great product/tool/project!

I was just testing out the new ACL functionality and it seems that only ACL:s with pattern type set to `LITERAL` are showing.

Please compare the kafka-acls.sh cli output below with the attached screenshot from Kowl where principal `Group:o=test3` is visible in the cli but not Kowl:

```
[root@kafka bin]# ./kafka-acls.sh --bootstrap-server kafka:9093 --command-config /opt/kafka/config/cli/kafka-console.properties --list
Current ACLs for resource `ResourcePattern(resourceType=TOPIC, name=test.*, patternType=LITERAL)`:
        (principal=Group:o=test2, host=*, operation=READ, permissionType=ALLOW)
        (principal=Group:o=test2, host=*, operation=DESCRIBE, permissionType=ALLOW)

Current ACLs for resource `ResourcePattern(resourceType=GROUP, name=group-1, patternType=LITERAL)`:
        (principal=User:cn=test,ou=test,o=test, host=*, operation=READ, permissionType=ALLOW)
        (principal=Group:o=test, host=*, operation=READ, permissionType=ALLOW)
        (principal=user:alice, host=*, operation=READ, permissionType=ALLOW)

Current ACLs for resource `ResourcePattern(resourceType=GROUP, name=group1, patternType=LITERAL)`:
        (principal=Group:o=test2, host=*, operation=READ, permissionType=ALLOW)

Current ACLs for resource `ResourcePattern(resourceType=TOPIC, name=test.*, patternType=PREFIXED)`:
        (principal=Group:o=test3, host=*, operation=DESCRIBE, permissionType=ALLOW)
        (principal=Group:o=test3, host=*, operation=READ, permissionType=ALLOW)

Current ACLs for resource `ResourcePattern(resourceType=TOPIC, name=test-topic, patternType=LITERAL)`:
        (principal=user:alice, host=*, operation=DESCRIBE, permissionType=ALLOW)
        (principal=User:cn=test,ou=test,o=test, host=*, operation=READ, permissionType=ALLOW)
        (principal=Group:o=test, host=*, operation=DESCRIBE, permissionType=ALLOW)
        (principal=User:cn=test,ou=test,o=test, host=*, operation=DESCRIBE, permissionType=ALLOW)
        (principal=user:alice, host=*, operation=READ, permissionType=ALLOW)
        (principal=Group:o=test, host=*, operation=READ, permissionType=ALLOW)

Current ACLs for resource `ResourcePattern(resourceType=GROUP, name=group1, patternType=PREFIXED)`:
        (principal=Group:o=test3, host=*, operation=READ, permissionType=ALLOW)
```
![image](https://user-images.githubusercontent.com/734196/98777668-73186080-23f1-11eb-969a-cecac15dee56.png)

**Comment by weeco** [comment]
Hey @atoom ,
thanks for reporting this! I'm able to reproduce this with our test clusters. Prefixed ACLs have been introduced in KIP-290 (released in Kafka 2.0.0). Generally sarama (our used Kafka library) is not really handy if we want to support such features while remaining backwards compatible for users who don't run v2.0.0. For this reason we'll be migrating to another Kafka client library (https://github.com/twmb/franz-go) which is superior to Sarama in many ways.

I'm working on this as we speak. This won't make it into the 1.2 release, but since you are already running on master I think there's a good chance that this will be available in 1-2 weeks. I'll leave this issue open and close it once this is merged into master :-).

**Comment by weeco** [comment]
@atoom It took longer than expected, because I waited for a few more changes in the new kafka library but I just pushed it without these changes into master. If you want to please give it a try and see if it is fixed for you.

**Comment by atoom** [comment]
@weeco Sorry for the late reply! I just tested and now I can see all the ACLs. However it seems that the `Pattern` column is not properly populated ? I would expect to see either `Literal` / `Prefixed`  / `*` but it is completely blank.

**Comment by rikimaru0345** [comment]
sorry for the delay, ill take a look

**Comment by rikimaru0345** [comment]
should be fixed in 88506f0
